src/tools/Connect/AzureStack.Connect.psm1

```diff
@@ -0,0 +1,148 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# See LICENSE.txt in the project root for license information.
```

Comment:
Lets not include the connect module. It has only VPN related to ASDK.
This script can be moved later to doc